PtyLab/utils/visualisation.py

```diff
@@ -67,7 +67,8 @@ def complex2rgb(u, amplitudeScalingFactor=1, force_numpy=True, center_phase=Fals
         amplitudeScalingFactor = ASF
     else:
         ASF = amplitudeScalingFactor
-    if ASF != 1:
+    
+    if ASF != 1 and amplitudeScalingFactor != "2sigma":
```

Comment:
Is this "2sigma" documented somewhere?

Reply:
If you traverse couple lines up, you will see that the condition "2sigma" seems to add a standard deviation. This condition however, is not handled properly in the following lines. Python assumes in this "else" case that amplitudeScalingFactor is a string "2sigma" and tries to multiply it with a numerical value (which is clearly incorrect). This fix simply separates that.